frontend/src/scenes/trends/PersonsModal.tsx

```diff
@@ -200,28 +199,37 @@ export function PersonsModal({
                         {people && (
                             <>
                                 {!!people.crossDataset?.length && people.seriesId !== undefined && (
-                                    <div className="data-point-selector">
-                                        <Select value={people.seriesId} onChange={(_id) => switchToDataPoint(_id)}>
-                                            {people.crossDataset.map((dataPoint) => (
-                                                <Select.Option
-                                                    value={dataPoint.id}
-                                                    key={`${dataPoint.action?.id}${dataPoint.breakdown_value}`}
-                                                >
-                                                    <InsightLabel
-                                                        seriesColor={getSeriesColor(dataPoint.id)}
-                                                        action={dataPoint.action}
-                                                        breakdownValue={
-                                                            dataPoint.breakdown_value === ''
-                                                                ? 'None'
-                                                                : dataPoint.breakdown_value?.toString()
-                                                        }
-                                                        showCountedByTag={showCountedByTag}
-                                                        hasMultipleSeries={hasMultipleSeries}
-                                                    />
-                                                </Select.Option>
-                                            ))}
-                                        </Select>
-                                    </div>
+                                    <LemonSelect
+                                        fullWidth
+                                        className="mb-2"
+                                        value={people.seriesId.toString()}
+                                        onChange={(_id) =>
+                                            typeof _id === 'string' ? switchToDataPoint(parseInt(_id, 10)) : null
+                                        }
+                                        options={
+                                            people.crossDataset.reduce(
+                                                (acc, dataPoint) => ({
+                                                    ...acc,
+                                                    [`${dataPoint.id}`]: {
```

Comment:
@pauldambra also hint for LemonSelect work - if the keys are numbers then all sorts of things don't work.